protoc-gen-bq-json-schema/genjson/config.go

```diff
@@ -3,7 +3,8 @@ package genjson
 import "github.com/spf13/pflag"
 
 type Config struct {
-	Format bool
+	Format                   bool
+	UseModeFromFieldBehavior bool
```

Comment:
Should we use  `SchemaOptions` here to ensure that the JSON schema supports the same options as the regular schema?

Reply:
Yeah, I don't see why not. However, with the addition of new `SchemaOptions` fields, it's still required to manually map CLI flags to them. But I guess that's fine. I'll add it.